src/app.js

```diff
@@ -439,7 +289,7 @@ function startRtcstatsServer() {
     logger.info('[App] Initializing: %s; version: %s; env: %s ...', appName, appVersion, getEnvName());
 
     setupWorkDirectory();
-    setupDumpStorage();
+    orphanFileHelper.processOldFiles();
```

Comment:
Will this run only once at the beginning? If that's the case we might have a problem with dumps which have not yet expired at this point. i.e. if a client decides not to reconnect to one of them and `processOldFiles ` doesn't run anymore they'll be orphaned.